Source/WebGPU/WebGPU/Device.h

```diff
@@ -233,7 +233,9 @@ class Device : public WGPUDeviceImpl, public ThreadSafeRefCountedAndCanMakeThrea
         std::optional<Error> error;
         const WGPUErrorFilter filter;
     };
-
+private PUBLIC_IN_WEBGPU_SWIFT:
+    id<MTLFunction> m_function;
```

Comment:
Can we rename to `m_nopVertexFunction` or similar please? When it was a function local variable the context was more obvious than its new position as a member variable.